packages/editor/src/managers/clipboard/paste-manager.ts

```diff
@@ -197,8 +197,14 @@ export class PasteManager {
         selectedBlock?.text?.insertList(insertTexts, endIndex);
         selectedBlock &&
           this._addBlocks(blocks[0].children, selectedBlock, 0, addBlockIds);
-
-        parent && this._addBlocks(blocks.slice(1), parent, index, addBlockIds);
+        //This is a temporary processing of the divider block, subsequent refactoring of the divider will remove it
```

Comment:
A tiny style suggestion: use `// hello` instead of `//hello`, adding a space after the double slashes. @JimmFly